### PR TITLE
updated doc issue OEIIITB-16

### PR DESCRIPTION
**Pull request by Sumith-Hegde**
Updated the docs for issue [OEIIITB-16](https://issues.redhat.com//browse/OEIIITB-16)

Version(s):

cherry pick version: GitOps 1.14, GitOps 1.13

Issue:
https://issues.redhat.com/browse/OEIIITB-16

Link to docs preview:
 https://84081--ocpdocs-pr.netlify.app/openshift-gitops/latest/accesscontrol_usermanagement/configuring-sso-for-argo-cd-using-keycloak.html#modifying-keycloak-resource-requests-limits_configuring-sso-for-argo-cd-using-keycloak

QE review:
- [x] QE has approved this change.

Additional information:

### CODE REVIEW

**Review by anandf — APPROVED** [review]
LGTM

**Review by Srivaralakshmi — COMMENTED** [review]
@Sumith-Hegde Also, on L15, we must change the CPU value column in the table to ensure correct labels are added to units.

from 500 to 500m

This changes must be made in the "gitops-docs-main" branch, which is the current GitOps 1.14 branch, and must be cherry-picked to the GitOps 1.13 branch. Copy and paste the changes and verify them in the preview.  This information must be added to the PR and visible to the merge reviewer.

**Comment by Srivaralakshmi** [comment]
@Dhruv-Soni11 Please peer review this PR and see that this PR meets all the requirements including the QE reviews (you must tag the QE person in the PR), before it goes to the peer review squad. Thank you!

**Comment by Dhruv-Soni11** [comment]
As mentioned by @Srivaralakshmi in the previous comment, I do not see a change from "500" to "500m" as mentioned in the description of the JIRA ticket. I see the change from "admin" to "administrator" in the preview link. @Sumith-Hegde could you ensure that you make this change and update the PR?

File: modules/gitops-modifying-keycloak-requests-limits.adoc [hunk 1]
```diff
@@ -27,5 +27,5 @@ $ oc -n openshift-gitops patch argocd openshift-gitops --type='json' -p='[{"op":
 
 [NOTE]
 ====
-Keycloak created by the {gitops-title} only persists the changes that are made by the operator. If the Keycloak restarts, any additional configuration created by the Admin in Keycloak is deleted.
+Keycloak created by the {gitops-title} only persists the changes that are made by the operator. If the Keycloak restarts, any additional configuration created by the administrator in Keycloak is deleted.
```
Comment (Dhruv-Soni11) [inline modules/gitops-modifying-keycloak-requests-limits.adoc:30]: Please also include change the following phrase to ensure better readability:

"the {gitops-title}" -> "the {gitops-title} Operator"

**Comment by Dhruv-Soni11** [comment]
> @Dhruv-Soni11 Please peer review this PR and see that this PR meets all the requirements including the QE reviews (you must tag the QE person in the PR), before it goes to the peer review squad. Thank you!

Hi @Srivaralakshmi I don't have permission to update the preview link in the PR description. This change can only be made by @Sumith-Hegde.

File: modules/gitops-modifying-keycloak-requests-limits.adoc (hunk 1 above)
Comment (Dhruv-Soni11) [reply modules/gitops-modifying-keycloak-requests-limits.adoc:30]: Also change "made by the operator" to "made by the Operator"

**Comment by Sumith-Hegde** [comment]
I have made the mentioned changes and updated the PR description with cherry pick version.

**Comment by Dhruv-Soni11** [comment]
Hello @Sumith-Hegde, thanks for making the changes in the content and updating the PR description. Both of these changes look good. I will ensure that these changes move to the QE review round now. Great work.

**Comment by varshab1210** [comment]
LGTM

**Review by skopacz1 — APPROVED** [review]
LGTM

**Pull request merged**